prow/config/jobs/metal3-io/baremetal-operator.yaml

```
@@ -86,6 +86,20 @@ presubmits:
         - make
         image: quay.io/metal3-io/basic-checks:golang-1.24
         imagePullPolicy: Always
+  - name: generate-api-docs
+    branches:
+    - main
+    run_if_changed: '(config/base/crds/bases/.*)$*'
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - --resources=config/base/crds/bases/
+        - --output=docs/api.md
+        command:
+        - crdoc
+        image: ghcr.io/fybrik/crdoc:latest
```

Must be pinned to a sha with tag in comment. For example:

```suggestion
        image: ghcr.io/fybrik/crdoc:shashashasha # v0.6.2
```